docs/troubleshooting-guide/my-dashboard-is-slow.md

```diff
@@ -4,133 +4,74 @@ title: My dashboard is slow
 
-1. Move cards from this dashboard into other dashboards until this dashboard's performance is acceptable.
-2. Use [custom destinations][custom-destinations] to link cards in the main dashboard to the other dashboards.
+Caching takes less effort because it doesn't involve any changes to your schemas or databases. If you're ready to invest more resources into the root cause of dashboard performance issues, check out this list of common [schema and database optimizations](https://metabase.com/learn/administration/making-dashboards-faster#organize-data-to-anticipate-common-questions).
 
-Note: this is a good way to design dashboards even when there aren't performance issues. A dashboard with a hundred cards isn't just slow, it's also hard to understand. Breaking things up improves comprehension as well as performance. Please read [this article][faster-dashboards] for tips on making dashboards more performant, and [this one][bi-best-practices] for designing good dashboards in general.
+## Embedded dashboard is slow compared to original dashboard
 
-## Is the database overloaded by other traffic?
+1. Set up a [locked parameter to pre-filter your data](../embedding/signed-embedding-parameters.md#pre-filtering-data-in-a-signed-embed).
 
-**Root cause:** Metabase is usually not the only application using your database, and you may not be the only person using Metabase. If someone else has opened a dashboard that launches a couple of dozen long-running queries, everyone else may then have to wait until database connections become free.
+**Explanation**
 
-**Steps to take:**
+One of the easiest ways to make a question or dashboard run faster is to work with a smaller dataset. Your Metabase admin can apply automatic data limitations using things like [SSO](../people-and-groups/start.md#setting-up-single-sign-on-sso), [data permissions](../permissions/data.md), and [data sandboxing](../permissions/data-sandboxes.md).
 
-1. Check the performance logs of the database server or the machine it's running on often reveals that the real problem is caused by some third-party application.
-2. The next steps depend on what those other applications are, how frequently they are making queries, whether the database can be replicated or the load can be moved onto other systems, and so on. If some of the applications involved are primarily being used for batch processing (e.g., daily or weekly reports), you can also check when those jobs are scheduled to ensure that they don't overlap.
+However, a standalone embedded dashboard (a [signed embed](../embedding/signed-embedding.md)) won't log people into Metabase. This means that people viewing the signed embed won't be subject to the permissions and data restrictions set up by your admin. When someone loads a question or a dashboard in a signed embed, that question or dashboard will utilize a larger, unrestricted dataset rather than a smaller, permissions-limited dataset.
```

Comment:
```suggestion
When someone loads a question or a dashboard in a signed embed, however, that question or dashboard will query the full dataset (rather than a smaller dataset limited by permissions). Standalone, [signed embeds](../embedding/signed-embedding.md) don't require people to be logged in, and unauthenticated people viewing the signed embed won't be subject to the permissions and data restrictions set up by your admin. 
```